modules/database/src/adapters/DatabaseAdapter.ts

```diff
@@ -211,6 +214,27 @@ export abstract class DatabaseAdapter<T extends Schema> {
     return true;
   }
 
+  protected async storeDroppedFields(schema: ConduitDatabaseSchema) {
```

Comment:
fields may have been dropped in objects. Currently this checks only top level fields that have been removed.

Ex. I might have a field profie:{profilePicture: String, isPublic: boolean} and maybe the new field is profile:{isPublic:boolean, hasFriends: boolean}. So we should know that profilePicture has been removed.